src/server/scripts/Northrend/Naxxramas/boss_maexxna.cpp

```diff
@@ -165,23 +193,49 @@ class npc_webwrap : public CreatureScript
 
     struct npc_webwrapAI : public NullCreatureAI
     {
-        npc_webwrapAI(Creature* creature) : NullCreatureAI(creature) { }
+        npc_webwrapAI(Creature* creature) : NullCreatureAI(creature), victimGUID(ObjectGuid::Empty), visibleTimer(0) { }
```

Comment:
we usually don't initialize ObjectGuid in the member constructor list because it's initialized by default to ObjectGuid::Empty